src/store/modules/images.js

```diff
@@ -361,24 +361,32 @@ const actions = {
 
             // and the end to noon tomorrow
             queryEnd = siteDate
-            queryEnd.setHours (12, 0, 0, 0)
+            queryEnd.setHours (12-siteUserDifference, 0, 0, 0)
             queryEnd.setDate(siteDate.getDate() + 1);
+            //console.log("later than noon")        
```

Comment:
I think keeping the comments is ok, but maybe add a description above each block of comments saying why they're there (and in what circumstances can they be removed). Otherwise they will probably blend in as regular abandoned comments when we forget this PR.